docs/API.md

```diff
@@ -0,0 +1,251 @@
+<a name="module_openid-client-helper"></a>
+
+## openid-client-helper
+The main module.
+
+
+* [openid-client-helper](#module_openid-client-helper)
+    * [module.exports(param0)](#exp_module_openid-client-helper--module.exports) ⇒ <code>object</code> ⏏
```

Comment:
Why use `param0` everywhere here?

Reply:
To clarify, this whole file, with links and everything, is automatically generated from jsdoc comments. The exported function takes a single config/options/args/params unnamed object argument. The actual name, `param0`, is a default VSCode uses, so I thought it may be better than zero known convention. (An alternative would be something like `$0`).

Reply:
The function itself looks like

```javascript
module.exports = function ({
  issuerMetadata,
  clientMetadata,
  useOpenId = true,
  usePKCE = true,
  resources = {
    'https://': {
      scope: '*'
    }
  },
  useMasterAccessToken = false,
  sessionKey = 'session',
  useResourceIndicators = false,
  customize
}) {
  // . . . no reference to `param0` here, only to its properties.
}
```